src/lib/intents.js

```diff
@@ -0,0 +1,139 @@
+import { cozyFetchJSON } from './stack'
+
+/*********************************************
+This following code should never be changed here since it's must be the exact same than in cozy-client-js/src/intents.js
+Service creating functions have been removed since it's not used by the bar
+**********************************************/
```

Comment:
I'll take a look at it, let's keep it as it yet. Maybe we can split tje `intents` lib into commonizable parts, and so have a _cozy-client-js_ flavour, and a vanilla one?